docs/guides/universal-profile/interact-with-contracts.md

```diff
@@ -266,15 +208,19 @@ const EOA = new ethers.Wallet(PRIVATE_KEY).connect(provider);
 
 ### Send the execute calldata
 
-The final step is to pass the encoded calldata to the Key Manager. Since we are calling from a UP's controller address (with proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions)), the Key Manager will authorize and execute the transaction.
+The final step is to pass the encoded calldata to the Universal Profile. Since we are calling from a UP's controller address (with proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions)), the Key Manager will authorize the transaction.
```

Comment:
+ add link to to LSP6 page to explain what a controller is.

```suggestion
The final step is to pass the encoded calldata to the Universal Profile. Since we are calling from an EOA that is a controller on the UP, the Key Manager will authorize the transaction.
```